jenkins/scripts/bare_metal_lab/cleanup-lab.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
+  environment:
+    EPHEMERAL_CLUSTER: "{{ EPHEMERAL_CLUSTER }}"
+    EXTERNAL_VLAN_ID: "{{ EXTERNAL_VLAN_ID }}"
+  vars_files:
+    - default_vars/vars.yaml
```

Comment:
why do we need to set vars for cleanup, are not they already set in deploy-lab.yaml?